src/app/app.component.html

```diff
@@ -1,6 +1,12 @@
 <div class="row">
   <app-sidebar></app-sidebar>
-  <div class="column" id="main">
-    <router-outlet></router-outlet>
+
+  <div class="cell flex-container flex-dir-column">
+    <div class="column flex-child-auto" id="main">
+      <router-outlet></router-outlet>
+    </div>
+    <div class="column flex-child-shrink" id="footer">
```

Comment:
Move this div into the footer component HTML, so the resulting app.component.html looks like:

```html
<div class="row">
  <app-sidebar></app-sidebar>

  <div class="cell flex-container flex-dir-column">
    <div class="column flex-child-auto" id="main">
      <router-outlet></router-outlet>
    </div>
    <app-footer></app-footer>
  </div>
</div>

```